src/main/java/sg/ncl/MainController.java

```diff
@@ -112,6 +112,8 @@
     private static final String NO_PERMISSION_PAGE = "nopermission";
 
     private static final String EXPERIMENTS = "experiments";
+    private static final String EXPERIMENT_ID = "expId";
+    private static final String EXPERIMENT_NAME = "expName";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "EXPERIMENT_NAME" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)